pubspec.yaml

```diff
@@ -31,31 +31,34 @@ dependencies:
   flutter:
     sdk: flutter
 
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.8
   http: ^1.2.2
   dart_nostr: ^8.2.0
   bloc: ^8.1.4
   flutter_bloc: ^8.1.6
-  flutter_secure_storage: ^9.2.2
   qr_flutter: ^4.0.0
   heroicons: ^0.10.0
-  cryptography: ^2.7.0
-  hive: ^2.2.3
   crypto: ^3.0.5
-  hive_flutter: ^1.1.0
-  pointycastle: ^3.9.1
   convert: ^3.1.1
   shared_preferences: ^2.3.2
-  web_socket_channel: ^2.4.0
   equatable: ^2.0.5
-  encrypt: ^5.0.3
   logging: ^1.2.0
   local_auth: ^2.3.0
   google_fonts: ^6.2.1
   timeago: ^3.7.0
+  flutter_riverpod: ^2.6.1
+  bitcoin_icons: ^0.0.4
+  collection: ^1.18.0
+  elliptic: ^0.3.11
+
+  nip44:
```

Comment:
NIP-44 support was implemented as per issue #3 but bundled into a separate library as it has utility for other Nostr developers. This library can be transferred to the mostro project if desired